docs/src/tutorials/tutorial_morphing.md

```diff
@@ -0,0 +1,188 @@
+# **Tutorial 9:** Morphing `Javis` Objects   
+
+There are multiple ways to morph an object in Javis.
+
+- Using the older `morph_to(::Function)` method. Very nice at matching the shapes for morphing . Has limitations on what the morphing function should contain.
+- Using (New) `morph_to(::Object)` method. Any Object can be morphed to any other object using this method.
+- Using `morph_to_fn(::Function)` method. Similar to `morph_to(::Object)` but morphs to function instead. Can morph an object to a function that contains Luxor calls to draw what it should morphed into.
+- Specifying an Action with interpolating Animation along with `morph()`
+
+This tutorial will focus on the last three.
+
+## Morphing one object to another.
+
+Like other animations `morph_to(::Object)` is to be used with action. To learn more about Actions refer to [Tutorial 5](tutorial_5.md)
+Here is a simple code snippet on how to use `morph_to`
+```julia
+using Javis
+
+video = Video(500,500)
+nframes = 160 
+
+function circdraw(colo)
```

Comment:
Is there a reason why you use `colo` and not `color`?

Reply:
its kind of a force of habit ,
Although im not using `Colors` , when i do it to avoid name clashes since 
Colors brings in the `color` function into the namespace . 
Its safe to use colors here though